peerconnection.go

```diff
@@ -348,7 +348,7 @@ func (pc *PeerConnection) checkNegotiationNeeded() bool {
 	for _, t := range pc.GetTransceivers() {
 		// https://www.w3.org/TR/webrtc/#dfn-update-the-negotiation-needed-flag
 		// Step 5.1
-		// if t.stoping && !t.stopped {
```

Comment:
Should these be commented out? They were in the original PR, maybe we should just delete?

Reply:
according to the spec we should have a `stopping` property on a transceiver but we dont right now. so this was added prematurely. i'll leave for now and we can remove it separately